fitbenchmarking/parsing/tests/test_parsers.py

```diff
@@ -517,8 +524,8 @@ def test_parse_problem_file(self):
         """
         Tests the parse_problem_file method
         """
-        filename = Path(__file__).parent / "nist" / "basic.dat"
-        fitting_problem = parse_problem_file(filename, OPTIONS)
+        filename = os.path.join(os.path.dirname(__file__), "nist", "basic.dat")
+        fitting_problem = parse_problem_file(filename, OPTIONS)[0]
```

Comment:
```suggestion
        filename = Path(__file__).parent / "nist" / "basic.dat"
        fitting_problem = parse_problem_file(filename, OPTIONS)[0]
```

Again I prefer slowly changing the code to use `Path` over `os`.

Reply:
Oops, I think I missed the Path change when rebasing on this one! Good spot